test/runtime/main.ts

```diff
@@ -1,25 +1,56 @@
+import * as dotenv from 'dotenv';
+import * as env from 'env-var';
 import { BigNumber, ethers } from 'ethers';
 
 import { createZToken } from '../../src';
-import { setEnv } from '../shared/setupEnv';
+import { SupportedChainId } from '../../src/config';
 
-(global as any).XMLHttpRequest = require('xhr2');
+dotenv.config();
+
+const networkConfig = {
+  [SupportedChainId.GOERLI]: {
+    rpcUrl: env.get('GOERLI_RPC_URL').required().asString(),
+    network: SupportedChainId.GOERLI,
+    privateKey: env.get('PRIVATE_KEY').required().asString(),
+  },
+  [SupportedChainId.RINKEBY]: {
+    rpcUrl: env.get('RINKEBY_RPC_URL').required().asString(),
+    network: SupportedChainId.RINKEBY,
+    privateKey: env.get('PRIVATE_KEY').required().asString(),
+  },
+  [SupportedChainId.MAINNET]: {
```

Comment:
Nit: We should prevent users from running the test file on mainnet. Don't want someone to accidently deploy to mainnet

Reply:
Resolved